src/lib/SX127xDriver/SX127x.cpp

```diff
@@ -484,6 +516,14 @@ uint32_t ICACHE_RAM_ATTR SX127xDriver::GetCurrBandwidthNormalisedShifted() // th
 
   switch (currBW)
   {
+  #if defined(RADIO_SX1272)
+  case SX127x_BW_125_00_KHZ:
+    return 64;
+  case SX127x_BW_250_00_KHZ:
+    return 32;
+  case SX127x_BW_500_00_KHZ:
+    return 16;
+  #else
```

Comment:
Same comment here as above.

Im just looking at minimising if defs in the lib to help with readability.

Reply:
done